IntegrationTests/Api.cs

```diff
@@ -20,260 +20,228 @@
 /// Check out https://github.com/specklesystems/server for               ///
 /// more info on the server.                                             ///
 ////////////////////////////////////////////////////////////////////////////
-namespace IntegrationTests
-{
-  public class RemoteOps
-  {
-    public ServerInfo testServer;
-    public Account firstUserAccount, secondUserAccount;
-
-    public Client myClient;
-    public ServerTransport myServerTransport;
-
-    private string streamId = "";
-    private string branchId = "";
-    private string branchName = "";
-    private string commitId = "";
-    private string objectId = "";
-
-    [OneTimeSetUp]
-    public void Setup()
-    {
-      testServer = new ServerInfo { url = "http://127.0.0.1:3000", name = "TestServer" };
-
-      // set up some users
-      using (var client = new WebClient())
+namespace TestsIntegration
+{
+  public class Api
+  {
+    public ServerInfo testServer;
+    public Account firstUserAccount, secondUserAccount;
+
+    public Client myClient;
+    public ServerTransport myServerTransport;
+
+    private string streamId = "";
+    private string branchId = "";
+    private string branchName = "";
+    private string commitId = "";
+    private string objectId = "";
+
+    [OneTimeSetUp]
+    public void Setup()
+    {
+      testServer = new ServerInfo { url = "http://127.0.0.1:3000", name = "TestServer" };
+
+      firstUserAccount = Utils.SeedUser(testServer);
```

Comment:
@didimitrie seed user creation has been moved to a utils class, should have broken it down in multiple PRs, sorrry!